tests/Vue/Pages/Results.spec.js

```diff
@@ -22,6 +22,25 @@ describe('Results.vue', () => {
         },
     }
 
+    it('displays intro text and instructions button', () => {
+        const wrapper = mount(Results, { mocks });
+
+        const intro = wrapper.find('#about-description');
+        expect(intro.isVisible()).toBe(true);
+
+        const instructionsButton = wrapper.find('.instructions-button');
+        expect(instructionsButton.isVisible()).toBe(true);
+
+    });
```

Comment:
Usually we don't test for specific sections to be rendered, as this might make the tests too brittle (in case a class or id changes) which is really like testing function internals. This also doesn't really ensure us that these selectors actually contain the content we intend them to. So I think the test might be redundant (unusual for me to say, I know, and it also pains me to say so :confounded: )

Reply:
This is actually a result of my "test first" approach on the task, but you're right, we can easily live without this one.